APP/controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require("./model");
+
+exports.getTopics = (req, res) => {
+  selectTopics().then((topics) => {
+    res.status(200).send({ topics });
+  });
```

Comment:
Could do with a catch block here, uncaught errors can crash the whole server ! For now you can just console.log the error, unless you want to build some middleware now for later use.